sort_bs_rotated.py

```diff
@@ -0,0 +1,72 @@
+'''
+// Time Complexity : O(logN)
+// Space Complexity : O(1)
+// Did this code successfully run on Leetcode : Yes
+// Any problem you faced while coding this : Spent too much time, but not able to solve
+
+
+// Your code here along with comments explaining your approach
+1. Find rotateIndex 
+    - Go to middle , check whether elem in mid is greater than next if so next is pivot
+    - else
+        - discard left array if it was sorted [ mid is greater than left]
+        - discard right array if it was sorted 
+2. Based on rotateIndex , search for target in either left or right sub array
+'''
```

Comment:
Good! Please try to combine both where you don't have to call `find_rotate_index(0, n - 1)` method separately.